frontend/lib/norent/homepage.tsx

```diff
@@ -40,7 +42,7 @@ export const JumpArrow = (props: { to: string; altText: string }) => (
 export const BuildMyLetterButton = (props: { isHiddenMobile?: boolean }) => (
   <span className={classnames(props.isHiddenMobile && "is-hidden-mobile")}>
     <CenteredPrimaryButtonLink to={NorentRoutes.locale.letter.latestStep}>
-      Build my letter
+      <Trans> Build my letter </Trans>
```

Comment:
Huh is there a reason the text is surrounded by spaces on both sides?

Reply:
No reason! Fixing it now